docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md

```diff
@@ -0,0 +1,30 @@
+
+On November 2, 2021, as part of our efforts to improve Codacy Cloud and allow all our customers to benefit from a more seamless experience, we’re removing the support for legacy manual organizations. As part of this process, we'll automatically migrate your legacy organization to a new [synced organization](../../organizations/what-are-synced-organizations.md).
+
+To make sure that Codacy will continue to analyze your repositories please perform the following steps before November 2, 2021:
+
+1.  **If you're using GitHub** you must [install the Codacy GitHub App](https://github.com/apps/codacy-production/installations/new) on the GitHub organizations that contain your repositories so that Codacy has the [necessary permissions](../../getting-started/which-permissions-does-codacy-need-from-my-account.md) to analyze your code.
+
+1.  Make sure that your Git provider organization owners [belong to the **Administrators** team](../../organizations/manual-organizations/creating-and-managing-teams.md) of your legacy organization on Codacy.
+
+If you already have any synced organizations on Codacy, the migration process won't overwrite any data on those synced organizations, nor copy any repositories that you have in your legacy organization.
+
+!!! important
+    After the migration process ends on December 2, 2021, we'll delete any existing legacy organizations that we couldn't migrate.
+
+## How does this impact me?
+
+Synced organizations have several advantages, such as keeping the repositories and team member permissions in sync with the Git provider. Besides this, Codacy is actively developing new features and improvements for synced organizations only.
+
+However, the following scenarios may impact you:
+
+-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. To restore the analysis of your repositories you can wait until Codacy attempts to migrate your organization, or restore the analysis immediately by performing the migration yourself.
+
+-   **If we can't automatically migrate your legacy organization** Codacy will stop analyzing your repositories, and you'll lose access to your existing repositories on Codacy. Please make sure that you follow the steps above to guarantee that the migration is successful.
+
+-   **If you have repositories in more than one Git provider organization** the migration process will create a synced organization for each of your Git provider organizations since it will no longer be possible to include repositories from multiple Git provider organizations in the same Codacy organization.
+
+    If your legacy organization is on a paid plan, and since billing is applied to each organization individually, Codacy will apply your existing plan to the new synced organization with the most repositories and apply a trial plan with the duration of 30 days to the remaining synced organizations.
```

Comment:
Can we omit this section?

Reply:
I think we should keep it. This will be happening, it's just up to Success how it affects their billing (and we'll be reaching out to them with a personalised strategy).  Due to this,  I think we should keep it as it's still relevant as a reference point for those customers

Reply:
Yes, as we'll be reaching out to these customers in advance

Reply:
Thanks for the feedback @harrycochran. I already removed this part. :+1: